lib/email_downcaser.rb

```diff
@@ -0,0 +1,8 @@
+class EmailDowncaser
+  def self.dump(value)
+    value.downcase
+  end
+  def self.load(value)
+    value
+  end
+end
```

Comment:
Shall I change it to only downcase the part after the @ per email?
This method was being used in the Signature model, so I moved it out into a separate file so that I can reuse it for Sponsor model as well.


> On 1 May 2015, at 15:03, Andrew White <notifications@github.com> wrote:
> 
> In lib/email_downcaser.rb <https://github.com/alphagov/e-petitions/pull/14#discussion_r29502839>:
> 
> > @@ -0,0 +1,8 @@
> > +class EmailDowncaser
> > +  def self.dump(value)
> > +    value.downcase
> > +  end
> > +  def self.load(value)
> > +    value
> > +  end
> > +end
> Not sure why we're down casing email addresses - technically the part before the @ can be case sensitive.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/alphagov/e-petitions/pull/14/files#r29502839>.
>